cluster-state-service/internal/features/e2etasksteps/common_steps.go

```diff
@@ -48,7 +48,7 @@ func init() {
 		}
 	})
 
-	Before("@task", func() {
+	Before("@task|@stream-instances", func() {
```

Comment:
Mostly for resource allocation concerns. This test relies on a task actually hitting the running status which fires off a new instance event. I'm stopping all tasks so that it does not sit waiting for available resources on an instance with limited resources.